test/test_torch_distributed_fsdp_meta.py

```diff
@@ -0,0 +1,161 @@
+import torch
+import torch.nn as nn
+from torch_xla.distributed.fsdp import XlaFullyShardedDataParallel
+from torch_xla.distributed.fsdp.wrap import (
+    always_wrap_policy as always_wrap,)
+
+import torch.distributed as dist
+import torch_xla.distributed.xla_multiprocessing as xmp
+import torch_xla.core.xla_model as xm
+
+import sys
+
+_TORCHDISTX_AVAIL = True
+try:
+  from torchdistx import deferred_init
```

Comment:
Do we need to install torchdistx in our CI for this coverage?

Reply:
No, I see upstream FSDP also has similar issue that it is not tested in the CI for the coverage. With latest torch we need to build torchdistx from source + some changes to match pytorch, so it is hard to maintain the C if we add it.

Reply:
Sad. But I guess as long as you guys are testing it and we brand it as experimental feature. Then it should be fine.